src/third-party/highlight.js

```diff
@@ -1,4 +1,4 @@
-import highlightjs from 'highlight.js/lib/highlight';
+import highlightjs from 'highlight.js/lib/core';
```

Comment:
Want to upgrade highlight in DCC too while this is front of mind?

Reply:
Sure. And if tacking that on, might as well add Julia support.

Reply:
https://github.com/plotly/dash-core-components/pull/871